Dockerfile

```diff
@@ -26,7 +26,7 @@ RUN python -m pip install --upgrade pip
 RUN conda install -y -c conda-forge gdal=3.0.4
 
 # Setup GDAL_DATA directory, rasterio needs it.
-ENV GDAL_DATA=/opt/conda/lib/python3.6/site-packages/rasterio/gdal_data/
+ENV GDAL_DATA=/opt/conda/lib/python3.7/site-packages/rasterio/gdal_data/
```

Comment:
There were a few instances where we were still using 3.6, so I updated it to 3.7 across the board.